perfkitbenchmarker/linux_packages/cassandra_stress.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 PerfKitBenchmarker Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Installs the cassandra-stress tool.
+
+Cassandra homepage: http://cassandra.apache.org
+cassandra-stress documentation:
+https://docs.datastax.com/en/cassandra/2.1/cassandra/tools/toolsCStress_t.html
+"""
+
+import posixpath
+
+from perfkitbenchmarker.linux_packages import cassandra
+from perfkitbenchmarker import vm_util
+
+CASSANDRA_DIR = posixpath.join(vm_util.VM_TMP_DIR, 'cassandra')
+
+
+def YumInstall(vm):
+  """Installs cassandra-stress on the VM."""
+  cassandra.YumInstall(vm)
+
+
+def AptInstall(vm):
+  """Installs cassandra-stress on the VM."""
+  cassandra.AptInstall(vm)
+
+
+def JujuInstall(vm, vm_group_name):
+  if vm.isController is True:
```

Comment:
I think this is only ever called on controllers so you shouldn't need this check (correct me if I'm wrong)